sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -4555,20 +4557,20 @@ var setParentClass = function(child, parent) {
         return 'modf';
     };
 
-    glsBuiltinPrecisionTests.Modf.prototype.doApply = function(ctx, iargs) {
+    glsBuiltinPrecisionTests.Modf.prototype.doApply = function(ctx, iargs, variablenames) {
         var intPart;
         var func1 = function(x) {
-            intPart = Math.floor(x);
+            intPart = Math.trunc(x);
             return x - intPart;
         };
         var func2 = function(x) {
-            return Math.floor(x);
+            return Math.trunc(x);
         };
 
         var fracIV = tcuInterval.applyMonotone1p(func1, iargs.a);
         var wholeIV = tcuInterval.applyMonotone1p(func2, iargs.a);
 
-        iargs.b = wholeIV;
+        ctx.env.m_map[variablenames[1]] = wholeIV;
```

Comment:
Yes. I have the same consideration. In fact, for this specific bug, I can use 
        ctx.env.m_map["out1"] = wholeIV. 
The variablenames[1] is definitely to be "out1".  
But there are failures in some other builtinprecision related test files. I suppose some other failures may have the same issue, then we can use whatever variable names we want in those cases. Besides, passing all 4 arguments is not a heavy operation. WDYT? @zhenyao .